TYPES.md

```diff
@@ -64,12 +64,39 @@ Parameters are :
      - it's used to retrieve informations from selectors
    - **prefix** (optional) is added to actions to avoid some collisions when there are two reducers with the same name in two distincts paths
 
-### state
+### state (without defaultData)
 ```es6
 {
   data: { <key1>: <instance1>, <key2>: <instance2> },
   array: [<instance1>, <instance2>],
   keys: [<key1>, <key2>],
+  initialized: false,
+}
+```
+
+### state (with defaultData)
```

Comment:
This section is not about instantiating the state, but this is about the state result in redux.
So you can rollback this section and remove the new one (or move it elsewhere)